src/browser/components/error/servererror.tsx

```diff
@@ -18,14 +20,24 @@ function ServerError(props: ServerError.Props) {
     return (
         <div className='jpe-ServerError-body'>
             <div className='jpe-ServerError-content'>
-                <div className='jpe-ServerError-icon'></div>
-                <h1 className='jpe-ServerError-header'>Jupyter Server Not Found</h1>
-                <p className='jpe-ServerError-subhead'>We were unable to launch a Jupyter server, which is a prerequisite for JupyterLab Desktop. If Jupyter is installed as a python module, but the python executable is not in your PATH, specify the executable location below. Otherwise, try installing or updating Jupyter. The Jupyter notebook version must be 6.0.0 or greater.</p>
+                <div className='jpe-ServerError-icon'/>
+                <h1 className='jpe-ServerError-header'>Jupyter Server Initialization Failed</h1>
+                <span className='jpe-ServerError-description'>
+                    <p>
+                    Jupyter Server, which is a prerequisite for JupyterLab Desktop, did not initialize properly.<br/>
+                    This might be because of an issue with the selected environment. You can change environment or try setting path to the server using buttons below.<br/>
+                    </p>
+                    <p>
+                    If this does not seem right, please report this issue in the JupyterLab Desktop repository, providing the error details presented below:
+                    </p>
+                </span>
+                <pre className='jpe-ServerError-error'>{props.error.name}: {props.error.message}</pre>
                 <div className='jpe-ServerError-btn-container'>
                     <button className='jpe-ServerError-btn' onClick={props.launchFromPath}>CHOOSE PATH</button>
```

Comment:
`launchFromPath` is not actually applying the selected path anymore. we should remove `CHOOSE PATH` option from here which was supposed to do what `CHANGE ENVIRONMENT` does anyway.

Reply:
Removed in 7ebd7fa6658bc8352b317d3e1ce885f2adb7ae32. Also rebased so it should build smoothly now.